plone/app/layout/viewlets/sections.pt

```diff
@@ -19,7 +19,7 @@
 
       <div class="plone-collapse plone-navbar-collapse" id="portal-globalnav-collapse">
         <ul class="plone-nav plone-navbar-nav" id="portal-globalnav">
-          <navtree tal:replace="structure view/render_globalnav" />
+          <navtree tal:replace="structure python:view.render_globalnav()" />
```

Comment:
Just curious: is there any speedup in this?
Should we push this as a good practice?

Reply:
yes :) tales is way slower than direct python expressions. Also, tales does mask errors, ``python:`` dont.

Reply:
:+1: